doc/source/reference/arrays.classes.rst

```diff
@@ -92,12 +92,21 @@ NumPy provides several hooks that classes can customize:
    :class:`ndarray` will unconditionally return :obj:`NotImplemented`,
    so that your reverse methods will get called.
 
-   .. note:: If you subclass :class:`ndarray`, we strongly recommend
-      that you avoid confusion by neither setting
-      :func:`__array_ufunc__` to :obj:`None`, which makes no
-      sense for an array subclass, nor by defining it and also defining
-      reverse methods, which methods will be called by ``CPython`` in
-      preference over the :class:`ndarray` forward methods.
+   .. note:: If you subclass :class:`ndarray`:
+
+      - We strongly recommend that you avoid confusion by neither setting
+        :func:`__array_ufunc__` to :obj:`None`, which makes no sense for
+        an array subclass, nor by defining it and also defining reverse
+        methods, which methods will be called by ``CPython`` in
+        preference over the :class:`ndarray` forward methods.
+      - :class:`ndarray` defines its own :func:`__array_ufunc__`, which
+        corresponds to ``getattr(ufunc, method)(*inputs,
+        **kwargs)``. Hence, a typical override of :func:`__array_ufunc__`
+        should call ``super().__array_ufunc__(*inputs, **kwargs)`` (of
```

Comment:
Why call if you override it? Also, this form does it work with python 2.7 and, apart from that, there seems to be an argument as to whether super should be used at all.

I find this description pretty confusing. Perhaps keep it short and put an extended explanation with code example in `numpy/doc/subclassing.py`